src/main/java/com/adobe/epubcheck/bitmap/BitmapChecker.java

```diff
@@ -138,20 +139,14 @@ public ImageHeuristics getImageSizes(String imgFileName) throws
 
 
 
-      if (formatFromSuffix != null && formatFromSuffix.equals(formatFromInputStream)) {
+    if (formatFromSuffix != null && formatFromSuffix.equals(formatFromInputStream)) {
       // file format and file extension matches; read image file
-      
-      try {
-        BufferedImage image = ImageIO.read(tempFile);
-        if (image == null) {
-          report.message(MessageId.PKG_021, EPUBLocation.create(imgFileName));
```

Comment:
Note: this potentially removes a case where `PKG_021` was raised before. This might never have been raised in practice (only one test is related to `PKG_021`, and this error message is raised at various code paths), but this need to be carefully reviewed.